### ISSUE TITLE
Playing all tracks by an artist should shuffle songs

### ISSUE THREAD

**Issue by jamesshaw1987** [Tyzer34/plexMusicPlayer#26]
Up for discussion, but when I play everything by an artist I'd like all their songs to be shuffled rather than cycling through their albums in alphabetical order.

A separate intent ("shuffle [artist]"?) would be another option.

**Comment by Tyzer34** [comment]
I agree. Conform to how Alexa is playing back music from Spotify for example, it always cycles through songs, etc. in the same order, which gets quite annoying after some time. An automatic shuffle is indeed a good idea.
